### DISCUSSION TITLE
SIGBUS error while using multiprocessor training.

### DISCUSSION THREAD

**Discussion by shahzaibbaig123** [axolotl-ai-cloud/axolotl#1595] [Q&A]
I am using mistral model and finetuning it. My server has 4 gpus. My training works fine when I use one or two gpus. But it gives me error when I try to increase the number of process in my training command. The command I am using is this: 
accelerate launch --num_processes=3 -m axolotl.cli.train /data/axo_configs/mistral/extract/config.yml --deepspeed deepspeed_configs/zero1.json
Also, this is the command which i use for getting the image and make a podman container:
podman run -v /data:/data --device nvidia.com/gpu=all --security-opt=label=disable --rm -it axolotl
This is the error:
![image](https://github.com/OpenAccess-AI-Collective/axolotl/assets/58305524/6e650de8-9aaf-41de-ad24-1887f7a9d734)

**Comment by NanoCode012** [comment]
We've added a note on this to our faq a while back, but this could be due to deepspeed. 

> A: Try upgrading deepspeed w: `pip install -U deepspeed`
